topology/Brookhaven National Laboratory/BNL ATLAS Tier1/BNL-ATLAS.yaml

```diff
@@ -370,6 +370,43 @@ Resources:
       StorageCapacityMax: 16800
       StorageCapacityMin: 16800
       TapeCapacity: 27000
+  BNL_ATLAS_TAPE:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: 1e47726bf41f2128059de31cf7bc9c8d5dcadad6
+          Name: BNL Atlas Tier 1 support
+        Secondary:
+          ID: OSG1000638
+          Name: Ivan Glushkov
+      Security Contact:
+        Primary:
+          Name: BNL Atlas Tier 1 support
+          ID: 1e47726bf41f2128059de31cf7bc9c8d5dcadad6
+        Secondary:
+          Name: Robert Hancock
+          ID: OSG1000212
+    Description: Duplicated from BNL_ATLAS_SE and updated for TAPE REST API blacklisting 
+    FQDN: dcdoor-tape.usatlas.bnl.gov
+    ID: 51
```

Comment:
```suggestion
```
New resources do not need IDs.

Reply:
Nice! Thanks! I had no idea what ID is or how I should generate it...

Reply:
Removed!